### PR TITLE
[EOC-126] Display archive lists below active lists on dashboard

### PR DESCRIPTION
**Pull request by alemikolo**
chrome-capture (1)

### CODE REVIEW

File: client/src/modules/dashboard/index.jsx [hunk 1]
```diff
@@ -59,9 +65,27 @@ class Dashboard extends Component {
...
+    this.handleArchivedListsData();
+  };
+
+  handleArchivedListsData = id => {
+    const { areArchivedListVisible } = this.state;
+    const {
+      fetchArchivedListsMetaData,
+      removeArchivedListsMetaData
+    } = this.props;
+    !areArchivedListVisible
```
Comment (adamklepacz) [inline client/src/modules/dashboard/index.jsx]: as Marek said, no teranry when nothing is return 
do something like. And try start with positive condition first not negative `!`
```
const action = areAchivedListVisible ? removeArchivedListsMetaData : fetchArchivedListsMetaData
action();
```

File: client/src/modules/dashboard/index.jsx [hunk 2]
```diff
@@ -59,9 +65,27 @@ class Dashboard extends Component {
 
   hideDialog = () => this.handleDialogContext(null)();
 
+  handleArchivedListsVisibility = () => {
+    this.setState(({ areArchivedListVisible }) => ({
+      areArchivedListVisible: !areArchivedListVisible
+    }));
+    this.handleArchivedListsData();
```
Comment (marekrozmus) [inline client/src/modules/dashboard/index.jsx]: this method uses `areArchivedListVisible` from state. To be sure that value will be set in state already use `setState` callback param to call this method

File: client/src/modules/dashboard/index.jsx [hunk 3]
```diff
@@ -21,6 +26,7 @@ import { Routes } from 'common/constants/enums';
 
 class Dashboard extends Component {
   state = {
+    areArchivedListVisible: false,
```
Comment (marekrozmus) [inline client/src/modules/dashboard/index.jsx]: maybe change to simple `showArchivedLists`?

File: client/src/modules/dashboard/index.jsx [hunk 4]
```diff
@@ -59,9 +65,31 @@ class Dashboard extends Component {
 
   hideDialog = () => this.handleDialogContext(null)();
 
+  handleArchivedListsVisibility = () => {
```
Comment (adamklepacz) [inline client/src/modules/dashboard/index.jsx]: unecessary braces

**Pull request merged**